.travis.yml

```diff
@@ -2,4 +2,20 @@ language: javascript
 
 install: "pip install --user $(whoami) -r requirements.txt"
 
-script: ""
\ No newline at end of file
+script: ""
```

Comment:
Doc says:
A build on Travis CI is made up of two steps:
install: install any dependencies required
script: run the build script
Why does script not contain the following in some form?
./setup.sh
. virtualenv/bin/activate